src/Components/WeatherDetails/WeatherAdditionalDetailsItem.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import WeatherIcon from '../WeatherIcon/WeatherIcon';
+import { FadeInTop } from '../../common/Animations/Animations';
```

Comment:
can't it be resolved to ./common/animations/Animations ??

Reply:
I know what you mean but I need to more practice with that library